Extensions/DialogueTree/JsExtension.js

```diff
@@ -290,6 +290,21 @@ module.exports = {
       .getCodeExtraInformation()
       .setFunctionName('gdjs.dialogueTree.loadState');
 
+    extension
+      .addAction(
+        'ClearState',
+        _('Clear dialogue state'),
+        _(
+          'Clear dialogue state. This resets all dialogue state accumulated by the player choices. Use for when the player is starting a new game.'
```

Comment:
`Use for when` => typo?
"Use it when the player " or "Useful when"

Reply:
fixed now